src/binding.cc

```diff
@@ -265,14 +244,16 @@ class Client : public ObjectWrap {
     }
 
     static void cb_close(uv_handle_t *handle) {
-      HandleScope scope;
+      NanScope();
+
       Client *obj = (Client*) handle->data;
+
       TryCatch try_catch;
       Handle<Value> emit_argv[2] = {
-        close_symbol,
-        Local<Boolean>::New(Boolean::New(obj->had_error))
+        NanNew<String>("close"),
+        NanNew<Boolean>(obj->had_error)
       };
-      obj->Emit->Call(obj->handle_, 2, emit_argv);
+      obj->Emit->Call(NanObjectWrapHandle(obj), 2, emit_argv);
```

Comment:
All function calls (including the `TryCatch` usage and related things) can probably be converted to use nan's method(s) for calling functions to make things tidier.